tests/ui/exec-trace/det_vals.rs

```diff
@@ -0,0 +1,83 @@
+// Copyright Kani Contributors
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+
+// kani-flags: --debug --harness ovoa --gen-exec-trace
```

Comment:
Can you add a comment on what this is testing please? Maybe state that for now we only print the values that were extracted.